pkg/piecestore/rpc/client/api/api.go

```diff
@@ -0,0 +1,117 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package api
+
+import (
+	"errors"
+	"io"
+	"log"
+
+	"github.com/zeebo/errs"
+
```

Comment:
don't need this space. i wonder if we can get our linters to check this sort of thing